CONTRIBUTING.md

```diff
@@ -293,6 +286,42 @@ As a result, the rules are pretty simple. For each language, create a program
 that writes the string "Hello, World!" to standard output. Ideally, the solution
 should be as simple as possible.
 
+### Longest Common Subsequence Rules
+
+Given two arrays of numbers, find the longest common subsequence. For example, let's say we have the
+following pair of arrays:
+
+```
+A = [1, 4, 5, 3, 15, 6]
+B = [1, 7, 4, 5, 11, 6]
+```
+
+The longest common subsequence is `[1, 4, 5, 6]`. 
+
+Write a program which accepts two command line arguments--each list--and outputs the longest 
+common subsequence between the two lists. Input arguments should be in array notation:
+`[1, 2, 14, 11, 31, 7, 9]`. 
```

Comment:
I might change this to a comma separated string based on #512, so hold tight.

Reply:
Fwiw, I think we should definitely change it. Parsing this array notation would be a nightmare in languages that don't natively support it (okay, it's not **that** bad, but a comma separated list is definitely more convenient)

Reply:
Doing that now!